src/containers/CreateCourse/index.jsx

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+
+import CreateCoursePage from '../../components/CreateCoursePage';
+import { fetchOrganizations } from '../../data/actions/publisherUserInfo';
+import { createCourse } from '../../data/actions/courseInfo';
+
+const mapStateToProps = state => ({
+  publisherUserInfo: state.publisherUserInfo,
+  courseInfo: state.courseInfo,
+});
+
+function mapDispatchToProps(dispatch) {
+  return {
+    fetchOrganizations: () => {
+      dispatch(fetchOrganizations());
+    },
+    createCourse: (courseData) => {
+      dispatch(createCourse(courseData));
+    },
+  };
+}
```

Comment:
I believe you can just write this as:

```
const mapDispatchToProps = {
  createCourse,
  fetchOrganizations,
};
```

And redux will do this boilerplate for you.

Reply:
👍